### PR TITLE
Use the node-exclusive tags for node e2e jobs

### PR DESCRIPTION
**Pull request by yujuhong**
Dependency: https://github.com/kubernetes/kubernetes/pull/64130, https://github.com/kubernetes/kubernetes/pull/64125

/cc @dchen1107 @Random-Liu

### CODE REVIEW

File: testgrid/config.yaml [hunk 1]
```diff
@@ -3600,8 +3630,10 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gke-device-plugin-gpu-p100-beta
   - name: kops-aws-1.11
     test_group_name: ci-kubernetes-e2e-kops-aws-beta
-  - name: kubelet-1.11
+  - name: kubelet-1.11-conformance
```
Comment (krzyzacy) [inline testgrid/config.yaml]: this is not the conformance job?

Comment (yujuhong) [reply testgrid/config.yaml]: The name change is unnecessary. Removed :)

**Comment by yujuhong** [comment]
Waiting for the dependency to be merged

**Pull request merged**